packages/ts/src/components/bullet-legend/modules/shape.ts

```diff
@@ -50,52 +51,68 @@ export function updateBullets (
   container.each((d, i, els) => {
     const shape = getString(d, config.bulletShape, i) as BulletShape
     const color = getColor(d, colorAccessor, i)
```

Comment:
Looks like there's no need to create this variable and call `getColor` if `d.color` is defined

Reply:
But it does the right thing when `d.color` is _not_ defined also, which I'm not sure how to handle differently from this. Did you have a concrete change in mind?